CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Adds new processor to `Minfraud::Components::Payment`: `:cardknox`,
   `:creditguard`, `:credorax`, `:datacap`, `:dlocal`, `:onpay`, and
   `:safecharge`.
+* Adds `rule_label` to minFraud output `/disposition`.
+* Adds `was_3d_secure_successful` attribute to
+  `Minfraud::Components::CreditCard`.
```

Comment:
These could probably benefit from including a bit more information about them. For example, "Adds support for the /credit_card/was_3d_secure_successful input. This is available by setting the was_3d_secure_successful attribute on Minfraud::Components::CreditCard" or something like that.